scoring.js

```diff
@@ -0,0 +1,85 @@
+ // returns 0 when the position is not one of the ones laid out below.
+    // create  function that gives points based on the rules we've have been given
+    // writing code for QB passing and QB rushing that calculates the total points for QB
+const  calculateScore = (player) => {
+   switch (player.position) {
+       case 'QB':
+           return  qbScore(player)
+        case 'RB':
+            return rbScore (player)
+        case 'WR':
+            return wrScore (player)
+        case 'TE':
+            return teScore (player)
+         case 'K':
+            return kScore (player)
+        default:
+            return 0
```

Comment:
This should all be handled by your default case. The requirements said "return 0 for any unknown position." The use of "K" in the test is just an example of an unknown position, the value could have been anything.